woocommerce/woocommerce-android
# Update secret `gradle.properties` for `tracks-gradle` configuration options

Relates to: #5212


### Description
This PR updates secret `gradle.properties` to include new configuration options for plugin added in #5214 . I could include this in previous PR but I forgot how secrets work, hence this PR.

### Testing instructions

1. If you have custom `gradle.properties` configuration, please backup it.
2. Checkout this branch
3. Update `gradle.properties` - `./gradlew configureApply`
4. Make sure you can see `tracksEnabled = true` at the bottom.
